internal/util/diff/diff_test.go

```diff
@@ -153,13 +162,24 @@ func TestCommand_RunCombinedDiff(t *testing.T) {
 	assert.NotEqual(t, commit, commit2)
 
 	localPkg := filepath.Join(w.WorkspaceDirectory, g.RepoName)
-	err = fetch.Command{
-		RepoSpec: &git.RepoSpec{
-			OrgRepo: g.RepoDirectory,
-			Ref:     "refs/tags/v2",
-			Path:    "/",
+	err = os.MkdirAll(localPkg, 0700)
+	assert.NoError(t, err)
+
+	kf := kptfileutil.DefaultKptfile(g.RepoName)
+	kf.Upstream = &kptfilev1alpha2.Upstream{
```

Comment:
Might want to consider reducing verbosity somehow: `kptfilev1alpha2`package name is repeated a lot

Reply:
Yeah, it is a bit verbose, but it kinda follows the pattern used in kubernetes. Maybe we should standardize on `kfv1alpha2` as the prefix. The default import name of `v1alpha2` is a bit unfortunate, so we should use a consistent alias.